src/common/helpers.js

```diff
@@ -114,3 +114,25 @@ export const getMetadataFields = experiment =>
   SampleFieldMetadata.filter(field =>
     experiment.sample_metadata.includes(field.id)
   ).map(field => field.Header);
+
+// Takes in arrays of samples and experiments as formatted by the serializer
+// and turns them both into objects where the keys are experiment accession
+// codes
+export const formatSamplesAndExperiments = (data, samples, experiments) => {
```

Comment:
What about splitting this into two methods? `formatSamples` and `formatExperiments`